### PR TITLE
Initial push of foundation for Tardis with no styling

### PR DESCRIPTION
**Pull request by karmatosed**
This is an initial commit using Create Block theme as a foundation to start the collaboration for Tardis. What this has is:

- Foundation theme.
- Style folder

There will be issues created to work on specific aspects of this, it is just to start the development as per issue #164

### CODE REVIEW

File: tardis/styles/twentytwentyfour.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+	"$schema": "https://schemas.wp.org/wp/6.5/theme.json",
+	"version": 2,
+	"title": "TwentyTwentyFour",
```
Comment (MaggieCabrera) [inline tardis/styles/twentytwentyfour.json:4]: Not a blocker for this but the title might look bad on the UI if we write them like this

**Review by MaggieCabrera — APPROVED** [review]
Approving so we can get the ball rolling with this one

**Comment by carolinan** [comment]
My only concern is the trademark registered on the name TARDIS. Would it cover this?

**Comment by MaggieCabrera** [comment]
> My only concern is the trademark registered on the name TARDIS. Would it cover this?

We should investigate that, but it's not a blocker, so I'm merging this. Let's revisit that when the theme is ready for publishing

**Pull request merged**